content/_index.html

```diff
@@ -55,7 +55,7 @@ <h2>Do What I Mean. Good Docs. Clean Code. Sounds good?</h2>
         provide support for rotated monitors.
     </li>
     <li>
-        Use a tree as data structure. This allows for more flexible layouts than
+        Use a tree as a data structure. This allows for more flexible layouts than
```

Comment:
Thanks for your suggestion, but it’s not just “a data structure”, it’s the main datastructure that i3 uses to arrange windows. On the other hand, writing “the main data structure” is also not ideal, because then readers might have the question “are there other important data structures in i3?”. 

With the current phrasing, even though it might not be grammatically correct, neither of these incorrect associations is provoked.

Maybe you have a suggestion for an entirely different phrasing. Or maybe we just leave things as they are… :)